src/com/epam/javatrainings/annotation/validator/AnnotationTypeDetector.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.javatrainings.annotation.validator;
+
+import com.epam.javatrainings.annotation.annotation_processors.*;
+import com.epam.javatrainings.annotation.custom_annotations.*;
+import com.epam.javatrainings.annotation.model.CustomerDto;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AnnotationTypeDetector {
+  static Map<String, String> error = new HashMap<>();
```

Comment:
what will contain the error data if I would call the typeDetect method twice,
one time with a customer with an invalid name, and one with an invalid discountRate???

Reply:
I made a change! 
Now the error data will contain a message about the name and discountRate.